chaincode/compute_plan_dag.go

```diff
@@ -75,7 +75,7 @@ func (dag *ComputeDAG) sort() error {
 			current[i].Depth = depth
 			final = append(final, current[i])
 			if _, ok := IDPresents[current[i].ID]; ok {
-				return fmt.Errorf("compute plan error: Duplicate training task ID: %s", current[i].ID)
+				return errors.Conflict("compute plan error: Duplicate training task ID: %s", current[i].ID)
```

Comment:
This is a BadRequest? The request is invalid due to duplicate ids. What do you think?

Reply:
I would say it's debatable here because its still a conflict but I think BadRequest is more coherent with the rest so let's change it.

Reply:
Done